ollieraikkonen/Freelancer-hd-edition-install-script
# UTF-8 bom characters removal

This pull requests adds a new method that removes a BOM header from any file.

**Do not merge before testing!**

TODO: Add this code to the latest 0.4.1 snapshot: https://github.com/ollieraikkonen/Freelancer-hd-edition-install-script/tree/0.4.1-4 . Or create a 0.4.1-5 based on that branch.

### Testing
1. Install like normally and check if the game launches.
2. Repeat step 1, but first enable `Use Unicode UTF-8 for worldwide language support`. You can find the option by going to Run (Win + R), type in intl.cpl, go to Administrative, and click Change system locale...
